src/components/clinical_trial_list/ClinicalTrialAddEdit.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import { FormField, FormValidators } from 'src/components/forms/forms'
+import { ClinicalTrial } from 'src/types/model'
+import { validationFailed } from 'src/utils/darFormUtils'
+import { ValidationError } from 'src/pages/dar_application/FormValidationState'
+
+const defaultClinicalTrial: ClinicalTrial = {
+  clinicalTrialId: '',
+  studyId: '',
+  title: '',
+  registry: '',
+  identifier: '',
+  status: '',
+  sponsor: '',
+  startDate: '',
+  endDate: '',
+  interventionType: '',
+  description: '',
+  phase: '',
```

Comment:
Can we set this to an enum?  https://clinicaltrials.gov/api/v2/studies/enums has:

```
[
  {
    "value": "NA",
    "legacyValue": "Not Applicable"
  },
  {
    "value": "EARLY_PHASE1",
    "legacyValue": "Early Phase 1"
  },
  {
    "value": "PHASE1",
    "legacyValue": "Phase 1"
  },
  {
    "value": "PHASE2",
    "legacyValue": "Phase 2"
  },
  {
    "value": "PHASE3",
    "legacyValue": "Phase 3"
  },
  {
    "value": "PHASE4",
    "legacyValue": "Phase 4"
  }
]
```